### DISCUSSION TITLE
Adding Tag and Comment along with Highlight

### DISCUSSION THREAD

**Discussion by aminasharaf** [stephanrauh/ngx-extended-pdf-viewer#2341] [Q&A]
We have a use case that involves adding comments and tags to the existing highlight feature. Currently, we have not encountered any similar features. We are expecting two additional buttons (tag and comment) alongside the highlight button. When these buttons are clicked, an event should be emitted so that our application can capture and respond to it. It would be helpful if we receive these events.

**Comment by stephanrauh** [comment]
@Calixteman The "comment" annotation seems to be a popular feature - I've seen multiple feature requests in the past. Do you have any plans to add this? BTW, is it one of the standard annotations defined by the PDF file format?

**Reply by aminasharaf** [reply]
Is there any way to add two buttons to the floating menu (color picker) of the highlight feature?
